app/gateway/network-ports-firewall.md

```diff
@@ -1,25 +1,62 @@
 ---
-title: Network, ports, and firewall for {{site.base_gateway}}
+title: "{{site.base_gateway}} ports"
 content_type: reference
 layout: reference
 
 products:
     - gateway
 
+min_version:
+  gateway: '3.6'
```

Comment:
```suggestion
  gateway: '3.4'
```

Reply:
@lena-larionova Port 8007 only applies to version 3.6 or later, but all the rest apply to 3.4. Should I just make a note in the description of that port?

Reply:
Reference pages have a version dropdown, so you _should_ be able to use version tags here. Try using `{% gte %}`? I'm not sure if the versions tags are the same here as the docs site.